templates/admin/forms.html.twig

```diff
@@ -56,13 +56,25 @@
           </li>
         </ul>
       </td>
-      <td class="text-right">
+      <td>
         <a href="{{ path('embed_delivery_start', { hashid: form|hashid(12) }) }}" target="_blank">
-          <i class="fa fa-external-link"></i>
+          <i class="fa fa-external-link"></i>&nbsp;{% trans %}basics.view{% endtrans %}
```

Comment:
You can use [bootstrap spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/) instead of space.
```suggestion
          <i class="fa fa-external-link mr-2"></i>{% trans %}basics.view{% endtrans %}
```